sale_product_set_packaging_qty/models/product_set.py

```diff
@@ -31,6 +36,7 @@ def _compute_product_packaging_qty(self):
                 line.product_packaging_qty = 0
                 continue
             line.product_packaging_qty = line.quantity / line.product_packaging_id.qty
+            line.write(line._prepare_product_packaging_qty_values())
```

Comment:
You should not write in a compute

```suggestion
            line.update(line._prepare_product_packaging_qty_values())
```